src/styles/themes/chromeDark.js

```diff
@@ -1,7 +1,7 @@
 const theme = {
   BASE_FONT_FAMILY: 'Menlo, monospace',
   BASE_FONT_SIZE: '11px',
-  BASE_LINE_HEIGHT: '14px',
+  BASE_LINE_HEIGHT: 1.31,
```

Comment:
`14px` comes from the original chrome devtools styles. imo we'd better leave it be and this is themable.

Reply:
1.31 would still give 14px line height,  but with the benefit of automatically scaling when the font size changes.

Reply:
+1 for not having absolute value otherwise user will always have to change lineheight along with fontsize.  `1.31` will autoo-adjust lineheight as per fontsize.

Reply:
Makes sense 👍. However `BASE_LINE_HEIGHT` isn't actually used anywhere :) . You would actually want to change `TREENODE_LINE_HEIGHT`. Also, the computed font size is actually 12px (instead of 11px). So to get 14px line height, the unitless line height would need to be `14/12`. I've taken a look at Chrome, it is using `normal` now.

According to MDN, 
> Depends on the user agent. Desktop browsers (including Firefox) use a default value of roughly 1.2, depending on the element's font-family.

Reply:
Thanks! could you change dark theme's line height as well? seems that you pushed light only

Reply:
LGTM if this is fixed. Nice work!